### ISSUE TITLE
tslint is deprecated

### ISSUE THREAD

**Issue by Zamiell** [timostamm/protobuf-ts#291]
Greetings, and thanks for the excellent library.

I'm using @protobuf-ts/plugin for the first time and I'm noticing that it creates the following comment at the top of the file:

![image](https://user-images.githubusercontent.com/5511220/164676024-1a692383-b87d-497d-b44b-f639f02cac48.png)

Some things:

### 1) Comment Stacking

Is it a bug that there are commas instead of newlines between the comments? Naively, I would expect it to look like:

```ts
// comment 1
// comment 2
// comment 3
```

Instead of:

```ts
// comment 1,// comment 2,// comment 3
```

### 2) TSLint is Deprecated

TSLint has been deprecated [for over 3 years](https://blog.palantir.com/tslint-in-2019-1a144c2317a9), so the `// tslint:disable` should be replaced with `/* eslint-disable */`.

**Comment by Zamiell** [comment]
Oh, I'm noticing now that the "comment stacking" problem seems to be fixed by #289.

**Comment by timostamm** [comment]
Oh yes, that was a regression, sorry about that.

Agree on TSLint - this is going to be replaced by eslint in the next major release. In the meantime, you can set the plugin option.
